pwiz_tools/Skyline/Skyline.cs

```diff
@@ -76,6 +76,7 @@
 using pwiz.Skyline.ToolsUI;
 using pwiz.Skyline.Util;
 using pwiz.Skyline.Util.Extensions;
+using DatabaseOpeningException = pwiz.Skyline.Model.Irt.DatabaseOpeningException;

```

Comment:
This seems like an oversight. I don't see that you are actually using this exception type.